packages/core-v2/src/visual-model/visual-entity.ts

```diff
@@ -1,16 +1,133 @@
-export type Position = {
+import { Entity, EntityIdentifier } from "./entity-model/entity";
+
+/**
+ * Color in hexadecimal, must start with "#" character.
+ * Should be in lower case.
+ */
+export type Color = string;
```

Comment:
The color type is already defined in [visual model](https://github.com/mff-uk/dataspecer/blob/040f1037c68be1c9bf7b97b15eac9cf166e47ce8/packages/core-v2/src/visual-model/visual-model.ts#L16) as `HexColor`.